xml/obs_build_job_constraints.xml

```diff
@@ -3,276 +3,462 @@
+    "3.0" is the value. </para>
+   <para> As described for project-scoped conditionals, above,
+    <literal>#!BuildConstraint</literal> lines can be guarded with various
+    conditionals to limit their effect to certain architectures
+    or, e.g., multibuild flavors. </para>
+  <note><para>
+   Be careful when setting build constraints. The idea is to use them to
+   express minimum values for the various parameters, below which builds
+   are likely to fail. If you set a constraint too high, you risk reducing the
+   pool of compliant build workers down to a very low number, or even to zero.
+   See <xref linkend="_constraint_handling"/> for a discussion of how the OBS
+   handles these cases.
+  </para></note>
+  </sect2>
+ </sect1>
+ <sect1 xml:id="_constraint_syntax">
+  <title>Constraint syntax</title>
+  <para> This section describes the various constraint qualifiers and their
+   syntax. </para>
+  <sect2>
+   <title><literal>hostlabel</literal></title>
+   <para> The "hostlabel" qualifier is any string which can be assigned to
+    build workers when starting the <literal>bs_worker</literal> process. It can be
+    used to restrict a build to specific workers, for example to run benchmarks
+    in a reproducible way. This constraint can also be defined as a negative
+    definition using the <literal>exclude=true</literal> attribute. However,
+    keep in mind that <literal>hostlabel</literal> settings are not portable,
+    since the always specific to a given OBS instance and should therefore only
+    be used as a last resort in situations that cannot be addressed by any of
+    the other qualifiers. </para>
+   <para> Example for <literal>_constraints</literal> file:</para>
+    <screen>&lt;constraints exclude="false"&gt;
+   &lt;hostlabel&gt;benchmark_runner&lt;/hostlabel&gt;
 &lt;/constraints&gt;</screen>
    <para>Example for project configuration:  </para>
    <screen>Constraint: hostlabel benchmark_runner</screen>
-   <para>Example for rpm spec file or Dockerfile:</para>
+   <para>Example for RPM spec file or Dockerfile:</para>
    <screen>#!BuildConstraint: hostlabel benchmark_runner</screen>
- </sect1>
- <sect1>
-  <title><literal>sandbox</literal></title>
-  <para> Defines the sandbox which is used for the build job. Sandboxes are
-   chroot, Xen or KVM environments. There is also the virtual secure sandbox,
-   which allows building on Xen or KVM. This constraints may also be defined as
-   a negative definition via the <literal>exclude=true</literal> attribute. </para>
-  <para> Example for <filename>_constraints</filename> file:</para>
-   <screen>&lt;constraints exclude="true"&gt;
-  &lt;sandbox&gt;secure&lt;/sandbox&gt;
+  </sect2>
+  <sect2>
+   <title><literal>sandbox</literal></title>
+   <para> Defines the sandbox which is used for the build. The "sandbox" is
+    the virtual environment in which the build takes place. Each worker is
+    configured with a fixed sandbox type. The possible values are:
```

Comment:
Also add something that this constraint is more intended for the OBS admin and usually not to be used by a project or package maintainer.